src/python/grpcio/grpc/_server.py

```diff
@@ -278,6 +278,9 @@ def set_trailing_metadata(self, trailing_metadata):
             self._state.trailing_metadata = trailing_metadata
 
     def abort(self, code, details):
+        # treat OK like other invalid arguments: fail the RPC
+        if code == grpc.StatusCode.OK:
+            code = grpc.StatusCode.UNKNOWN
```

Comment:
Please add a log statement here warning the user they are doing they are not supposed to do.

Reply:
I've added it as `logging.warning`. I could not find any other places in gRPC Python that log anything other than `logging.exception`, so I hope that makes sense.